indico/modules/events/management/util.py

```diff
@@ -182,3 +183,38 @@ def _process_attachments(folders):
 
     else:
         raise TypeError('Unexpected object of type {}: {}'.format(type(root).__name__, root))
+
+
+def find_unregistered_users(person_list):
+    """Get a set of users which don't have an Indico account
+
+        :param person_list: List of persons to check
+
+    """
+    return set(filter(lambda p: p.person.user is None, person_list))
+
+
+def flash_if_unregistered(former_person_list, new_person_list, event):
+    """ Flash a message when user(s) with no Indico account is added for a contribution or as a chairperson
```

Comment:
this line is way too long for a docstring. try to limit it to 72 chars, if it needs to be longer start it in the next line (only start it in the `"""` line if it fits there).

also, too much whitespace